src/docs/product/performance/transaction-summary.mdx

```diff
@@ -12,6 +12,10 @@ Every transaction has a summary view that gives you a better understanding of it
 
 If you find an important transaction you want to return to, click "Key Transaction" in the upper right corner to star it. This will be saved so you can view this from your [Performance Homepage](/product/performance/). Keep in mind any key transaction you mark will not be shared with others in your organization and are limited to ten per user.
 
+## Web Vitals
+
+Frontend transactions will have a "Web Vitals" tab. Clicking on the tab will take you to the [Web Vitals page](/product/performance/web-vitals), where you can see a detailed view of the web vitals associated with this transaction.
```

Comment:
Doesn't the tab show up only if frontend transactions have any measurements?

Reply:
The goal is for it to show up for any FE transaction. If there's no web vitals we have an empty state that explains what they do, why they might not be there and to upgrade SDK if necessary